lib/game.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+require_relative 'board.rb'
+require_relative 'player.rb'
+# Game class
+class Game
+  include UI
+
+  attr_accessor :continue
+
+  def initialize(board, player1, player2)
+    @new_board = board
+    @player1 = player1
+    @player2 = player2
+  end
+
+  def play
```

Comment:
This method was not tested in your game_spec.rb

I am sure you can think of some ways of testing this method.

To consider project complete I must see that you have tested all public  methods of classes you have implemented in this project.


I'm sure you can think of some 😄! If not, I am here to help, simply contact me!

Reply:
The change for this method has been done